### ISSUE TITLE
Ability to be able to publish from google drive, google photos, onedrive and the likes

### ISSUE THREAD

**Issue by ceoger** [lbryio/lbry-android#958]
## The Issue
Unable to publish from google drive/one drive, there is a an error for file path could not be determined
### Steps to reproduce
1. Try publishing and select the option for file
2. Select one drive or gdrive
3. Select the content you want to publish, you get a black screens and an error file path could not be determined

### Expected behaviour
Users should be able to publish directly from one drive or gdrive without having to download the content

### Actual behaviour
file path could not be determined

## System Configuration
- Device:
- Android version:

## Screenshots

https://cdn.discordapp.com/attachments/440189329139171350/730141080988680343/Screenshot_20200708-004930_LBRY.jpg
### Acceptance Criteria
1.
2.
3.

### Definition of Done
- [ ]  Tested against acceptance criteria
- [ ] Tested against the assumptions of the user story
- [ ] The project builds without errors
- [ ] Unit tests are written and passing
- [ ] Tests on devices/browsers listed in the issue have passed
- [ ] QA performed & issues resolved
- [ ] Refactoring completed
- [ ] Any configuration or build changes documented
- [ ] Documentation updated
- [ ] Peer Code Review performed

**Comment by kekkyojin** [comment]
See #660

**Comment by akinwale** [comment]
@ceoger Was the file you tried to select already downloaded to your device locally?

**Comment by ceoger** [comment]
Nope, the file was not downloaded. It was in my gdrive, android allows you to navigate to select files from OneDrive
